packages/react/src/internal/__mocks__/mockHTMLElement.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const hep = HTMLElement.prototype;
+export const mockHTMLElement = (options: HTMLElement) => {
+  const originals = {};
```

Comment:
May be worth typing `originals`.

```suggestion
  const originals: Record<string, PropertyDescriptor | undefined> = {};
```